openedx_rocketchat/utils.py

```diff
@@ -0,0 +1,205 @@
+import hashlib
+import logging
+import re
+
+from django.conf import settings
+from django.core.cache import cache
+
+from openedx_rocketchat.edxapp_wrapper.get_configuration_helpers import get_configuration_helpers
+from openedx_rocketchat.edxapp_wrapper.get_student_models import anonymous_id_for_user, get_user
+
+from rocketchat_API.rocketchat import RocketChat as ApiRocketChat
```

Comment:
rocketchat_API it is a pip package? this need to be inside of requirements file. @andrey-canon